crates/ruff_python_parser/tests/snapshots/invalid_syntax@try_stmt_invalid_order.py.snap

```diff
@@ -0,0 +1,83 @@
+---
+source: crates/ruff_python_parser/tests/fixtures.rs
+input_file: crates/ruff_python_parser/resources/inline/err/try_stmt_invalid_order.py
+---
+## AST
+
+```
+Module(
+    ModModule {
+        range: 0..47,
+        body: [
+            Try(
+                StmtTry {
+                    range: 0..31,
+                    body: [
+                        Pass(
+                            StmtPass {
+                                range: 9..13,
+                            },
+                        ),
+                    ],
+                    handlers: [],
+                    orelse: [],
+                    finalbody: [
+                        Pass(
+                            StmtPass {
+                                range: 27..31,
+                            },
+                        ),
+                    ],
+                    is_star: false,
+                },
+            ),
+            Pass(
+                StmtPass {
+                    range: 42..46,
+                },
+            ),
+        ],
+    },
+)
+```
+## Errors
+
+  |
+3 | finally:
+4 |     pass
+5 | else:
```

Comment:
Do I understand this correctly that the `else` gets dropped as part of the statement error recovery code?

Reply:
Hmm, it won't be dropped but there will be additional error as then it's outside the `try` statement. So, the statements in the `else` body will be parsed as if it's outside the `try` statement. I think it should be trivial to recover from this by parsing it again.